corehq/apps/app_manager/tests/test_submodule_xform.py

```diff
@@ -0,0 +1,112 @@
+from mock import patch
+from django.test import SimpleTestCase
+from corehq.apps.app_manager.const import APP_V2
+from corehq.apps.app_manager.models import (
+    AdvancedModule,
+    Application,
+    FormActionCondition,
+    LoadUpdateAction,
+    Module,
+    OpenSubCaseAction,
+    UpdateCaseAction,
+)
+from corehq.apps.app_manager.tests import TestFileMixin
+
+
+class SubmoduleXFormBase(TestFileMixin):
```

Comment:
These should all go in the `test_child_module.py` file (which should perhaps be renamed to `test_submodule.py`)